src/extensions/ffxiv_weather.py

```diff
@@ -0,0 +1,67 @@
+from discord.ext import commands
+import discord
+
+import datetime
+import ffxivweather
+
+list = ["Limsa Lominsa", "Middle La Noscea", "Lower La Noscea", 
+    "Eastern La Noscea", "Western La Noscea", "Upper La Noscea", 
+    "Outer La Noscea", "Mist"]
+
+
+class FFXIVWeather(commands.Cog):
+    def __init__(self, bot):
+        self.client = bot
+
+    def __create_embed(self, zone, amount: int = 15):
```

Comment:
This function is named inaccurately and does too much on it's own. You should split this code up into at least 2 or 3 separately named functions.